CHANGELOG.md

```diff
@@ -4,6 +4,43 @@
 
 ### Compiler
 
+- The compiler can now suggest to wrap a value in an `Ok` or `Error` if that can
+  solve a type mismatch error:
+
+  ```gleam
+  pub fn first(list: List(a)) -> Result(a, Nil) {
+    case number {
+      [] -> Error(Nil)
+      [first, ..rest] -> first
+    }
+  }
+  ```
+
+  Results in the following error:
+
+  ```txt
+  error: Type mismatch
+    ┌─ /src/one/two.gleam:5:5
+    │
+  5 │     [first, ..rest] -> first
+    │     ^^^^^^^^^^^^^^^^^^^^^^^^
+    │                        │
+    │                        Did you mean to wrap this in an `Ok`?
```

Comment:
This changelog entry needs updating I think? It no longer makes this suggestion

Reply:
Updated, thank you!